src/api/app/models/project/update_from_xml_command.rb

```diff
@@ -1,6 +1,6 @@
 class Project
   class UpdateFromXmlCommand
-    include Project::Exceptions
```

Comment:
Same here, please squash it to the commit that introduced this. This also makes it easier to review the PR